Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -112,13 +112,16 @@ struct RDKIT_DISTGEOMHELPERS_EXPORT EmbedParameters {
   bool ignoreSmoothingFailures;
   bool enforceChirality;
   bool useExpTorsionAnglePrefs;
+  bool useSmallRingTorsions;
```

Comment:
I'm not a huge fan of adding arguments to the middle of constructors like what this change requires. When you do that, you force other developers to look through all the arguments in their code and figure out where they need to insert new values. If you put them at the end, this is super easy.
We can do it if we need to, but I'm wondering why it's necessary here.

Reply:
reordered to put varibles last, I am now also exposing `ETversion` in `EmbedMultipleConfs` and `EmbedMolecules` I hope this is okay.